### PR TITLE
Feature/fix pixel diff filter

### PR DESCRIPTION
**Pull request by enesoezel**

### CODE REVIEW

File: src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java [hunk 1]
```diff
@@ -15,6 +18,7 @@
 @Getter
 @Slf4j
 public class PixelDiffFilter implements Filter {
+	//TODO Move this web specific class to recheck-web
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java]: As it belongs to the class, I suggest to move it above the annotations. (And please insert a space: `// TODO ...` 🤓)

File: src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java [hunk 2]
```diff
@@ -25,6 +29,8 @@
 	private final boolean specifiedAsDouble;
 	private final double pixelDiff;
 
+	private static final Set<String> ignoredKeys = new HashSet<>( Collections.singletonList( "style" ) );
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java]: Should be moved above the instance fields, where also the other `static` field is.

File: src/test/java/de/retest/recheck/review/ignore/PixelDiffFilterTest.java [hunk 3]
```diff
@@ -87,4 +87,13 @@ void specifiedAsDouble_flag_should_control_toString() throws Exception {
 		assertThat( new PixelDiffFilter( true, 0.0 ) ).hasToString( "pixel-diff=0.0" );
 	}
 
+	@Test
+	void style_attribute_should_return_false() {
+		final String expected = "18px";
```
Comment (beatngu13) [inline src/test/java/de/retest/recheck/review/ignore/PixelDiffFilterTest.java]: You could use "real-world" test data:

```java
final String expected = "width: 1px; height: 1px";
final String actual = "width: 1000px; height: 1000px";
```

**Review by beatngu13 — CHANGES_REQUESTED** [review]
Could please also adapt the log? I don't think we need the `NumberFormatException` in there as it is quite ugly and we know what the issue is. Also it would be helpful to quote expected and actual:

```java
log.error( "Could not parse expected '{}' and actual '{}' for pixel diff.", expected, actual );
```

File: src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java (hunk 2 above)
Comment (diba1013) [reply src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java]: For now?
```suggestion
	private static final Set<String> ignoredKeys = Collections.singleton( "style" );
```

**Review by beatngu13 — APPROVED** [review]

**Pull request merged**